### DISCUSSION TITLE
Warring factions

### DISCUSSION THREAD

**Discussion by Jaz0314** [Vdauphin/HeartsAndMinds#1638] [Question and answer] [answered]
Is it possible to have two enemy factions who also fight each other? For example opfor is the first enemy faction and then have a second Indfor faction who are both hostile to you and each other. Is that something possible or would it break the game completely with the spawning?

**Comment by Vdauphin** [comment, marked as answer]
Hello,

> Is it possible to have two enemy factions who also fight each other?

Yes, you can spawn them via Zeus
If you want to go on this side, I would drop H&M to ALiVE. It will give you a better experience

> Is that something possible 

Yes

> or would it break the game completely with the spawning?

No

Cheers
